device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -548,6 +548,10 @@ public void subscribeToDeviceMethod(DeviceMethodCallback deviceMethodCallback, O
      *	      the service checking for availability of new messages. The value
      *	      is expected to be of type {@code long}.
      *
+     *	    - <b>SetSendInterval</b> - this option is applicable to all protocols.
+     *	      This value sets the period (in milliseconds) that this SDK spawns a thread to send a queued message.
```

Comment:
time instead of period?

Reply:
Period implies that it spawns periodically, so I like it here

Reply:
I see. I did not infer that initially but makes sense now.